myia/cconv.py

```diff
@@ -8,11 +8,11 @@
 from typing import Dict, Iterable, Optional, Set
 from collections import defaultdict
 
-from myia.anf_ir import ANFNode, Constant, Graph
-from myia.anf_ir_utils import \
+from .ir.anf import ANFNode, Constant, Graph
+from .ir.utils import \
```

Comment:
I vaguely remember pep8 not liking escaping the end of lines, and favoring parentheses.

Reply:
I have a slight preference for the backslashes and `flake8` didn't complain. I'm open to changing it, though, if that's what others prefer.

Reply:
`isort` offers seven different ways to handle long imports (scroll down on https://pypi.org/project/isort/). I'd be fine with either 2 or 5. Thoughts?